### DISCUSSION TITLE
Link in the sidebar

### DISCUSSION THREAD

**Discussion by manfahrer** [hassio-addons/addon-uptime-kuma#25] [Q&A] [answered]
Hello would it be possible that an automatic link is generated and displayed in the sidebar.

**Comment by frenck** [comment, marked as answer]
No, that is not possible. This is not an add-on limition, but a technical limitation of Uptime Kuma itself.

**Reply by tomber71** [reply]
Will this make it possible?
https://github.com/hassio-addons/app-uptime-kuma/discussions/289
